source/fundamentals/indexes.txt

```diff
@@ -396,6 +396,25 @@ For more information, see the :manual:`Unique Indexes</core/index-unique>` page
 
 .. driver-content-end
 
+Wildcard Indexes
+~~~~~~~~~~~~~~~~
+
+.. _wildcard-indexes:
+
+Wildcard indexes enable queries against unknown or arbitrary fields. 
+These indexes can be beneficial if you are using a dynamic schema.
+
+The following example creates an ascending wildcard index on all 
+values of the ``location`` field, including values nested in subdocuments and arrays:
```

Comment:
Issue:

I could be wrong, but I think the terminology is usually used when creating an index is to specify that it is on the field rather than the value. For example, see the headings on the [createIndex](https://www.mongodb.com/docs/manual/reference/method/db.collection.createIndex/#examples) Server Manual page. Each of the headings mentions "Create an Index on... Field". The action MongoDB takes is to index the values for the fields.

Suggestion:

```suggestion
The following code example shows how you can create an ascending
wildcard index to instruct MongoDB to index values of the \`\`location\`\`
field, and all values nested in subdocuments and arrays:
```